docker-images/kafka/scripts/kafka_config_generator.sh

```diff
@@ -3,6 +3,7 @@
 # Write the config file
 cat <<EOF
 broker.id=${KAFKA_BROKER_ID}
+broker.rack=${KAFKA_RACK}
```

Comment:
Does Kafka accept it when `KAFKA_RACK` is unset (because the user didn't bother setting up this feature)? Presumably so because you've tested it.

Reply:
Yeah it works fine ;)

Reply:
It might work, but it will look completely ugly. I think this should be done cleaner.

Reply:
I don't see it as a problem. In any case, the configuration file is something that we and users don't have to manipulate; other than that, the broker startup component is able to handle fields without a value in the properties file.

Reply:
Well ... I think it looks quite strange when you see in a config file something like:

```
something=
```

Reply:
I see some files with things like this but who is going to take a look at the config file ... should be the Strimzi role to hide all the mess around that ?

Reply:
In this way you can see if your rack configuration was got or not quite immediately :-)